samples/subsys/usb/cdc_acm/src/main.c

```diff
@@ -118,7 +118,12 @@ void main(void)
 	uart_irq_rx_enable(dev);
 
 	/* Echo the received data */
+	int i = 0;
```

Comment:
I think you should put declaration in the beginning of the function, or use `for()` loop declaring it.

Reply:
Thank you for the reminder. 
Done as requested. I moved declaration to top